libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Steps/BaseInvokeDialog.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Steps
     /// <summary>
     /// Step which calls another dialog
     /// </summary>
-    public abstract class BaseInvokeDialog : Dialog, IDialogDependencies
+    public abstract class BaseInvokeDialog : DialogCommand
```

Comment:
As I mentioned... I dropped DialogCommand as a class and just added an "inheritState" property to Dialog.